src/components/App.tsx

```diff
@@ -1153,7 +1153,9 @@ class App extends React.Component<AppProps, AppState> {
   // Copy/paste
 
   private onCut = withBatchedUpdates((event: ClipboardEvent) => {
-    if (isWritableElement(event.target)) {
+    const isExcalidrawActive =
+      document.activeElement === this.excalidrawContainerRef.current;
```

Comment:
Let's instead do `this.excalidrawContainerRef.current.closest(document.activeElement)` to catch edge cases where the focus is on some child.

Reply:
you meant `this.excalidrawContainerRef.current.contains(document.activeElement)` ? 
Though it might not be useful since we prevent selection outside the canvas.

Reply:
yes, sorry, I meant `contains` indeed. I'd rather use that coz then we won't have to hunt for edge cases later. For example when we programmatically focus elements other than the container. To my knowledge using the `contains` workflow won't have any drawbacks.